addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-linux-instance.cfn.yml

```diff
@@ -173,6 +187,10 @@ Resources:
           DeviceIndex: '0'
           GroupSet:
             - !Ref SecurityGroup
+            - !If
+              - AppStreamEnabled
+              - Fn::ImportValue: !Sub "${SolutionNamespace}-SwbVPCDefaultSG"
```

Comment:
Yep, that's right. Since default security group has ingress to itself other EC2 instances have connectivity to each other. However, since those instances still require ssh key, those instances are still secured.